### PR TITLE
Cafcass & Pathfinder WA Task changes

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-6155
https://tools.hmcts.net/jira/browse/PRL-5951
https://tools.hmcts.net/jira/browse/PRL-6191
https://tools.hmcts.net/jira/browse/PRL-6262

### Change description ###
WA path finder, close, reopen cases

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json [hunk 1]
```diff
@@ -5581,6 +5581,58 @@
...
+        "UserRoles": [
+          "caseworker-privatelaw-courtadmin",
+          "caseworker-privatelaw-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-wa-task-configuration",
+          "caseworker-privatelaw-solicitor",
```
Comment (shashisk97) [inline definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json:5626]: Solicitor is needed?

File: definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json [hunk 2]
```diff
@@ -5581,6 +5581,58 @@
...
```
Comment (shashisk97) [inline definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json:5600]: Solicitor is needed?

File: definitions/private-law/json/CaseEventToFields/Pathfinder/CaseEventToFields.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "pathfinderDecision",
+    "CaseFieldID": "isPathfinderCase",
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Yes",
+    "Publish": "True"
```
Comment (sahaSonali) [inline definitions/private-law/json/CaseEventToFields/Pathfinder/CaseEventToFields.json:12]: Why need to publish this filed?

File: definitions/private-law/json/CaseField/Pathfinder/CaseField.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "isPathfinderCase",
+    "Label": "Is this a Pathfinder case?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
```
Comment (sahaSonali) [inline definitions/private-law/json/CaseField/Pathfinder/CaseField.json:8]: Add searchable N

**Review by shashisk97 — APPROVED** [review]

**Pull request merged**